### ISSUE TITLE
Rename file example

### ISSUE THREAD

**Issue by sayedihashimi** [dotnet/dotnet-template-samples#8]
Add a sample showing how to rename a file

**Comment by neman** [comment]
Is this related to https://github.com/dotnet/templating/wiki/%22Runnable-Project%22-Templates#sourcesrename-optional?

**Comment by sayedihashimi** [comment]
@neman yes. More info on how to use it in comments at https://github.com/dotnet/templating/issues/1238
